public/app/features/api-keys/ApiKeysForm.tsx

```diff
@@ -75,18 +87,9 @@ export const ApiKeysForm: FC<Props> = ({ show, onClose, onKeyAdded }) => {
               <Input type="text" className="gf-form-input" value={name} placeholder="Name" onChange={onNameChange} />
             </div>
             <div className="gf-form">
-              <span className="gf-form-label">Role</span>
-              <span className="gf-form-select-wrapper">
-                <select className="gf-form-input gf-size-auto" value={role} onChange={onRoleChange}>
-                  {Object.keys(OrgRole).map((role) => {
-                    return (
-                      <option key={role} label={role} value={role}>
-                        {role}
-                      </option>
-                    );
-                  })}
-                </select>
-              </span>
+              <InlineField label="Role">
+                <Select inputId="role-select" value={role} onChange={onRoleChange} options={ROLE_OPTIONS} />
```

Comment:
```suggestion
                <Select inputId="role-select" value={role} onChange={onRoleChange} options={ROLE_OPTIONS} menuShouldPortal />
```
I think this needs `menuShouldPortal` prop to get rid of deprecation warning in console.